oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/LogoutHandlerTest.java

```diff
@@ -55,10 +50,7 @@
 import static org.mockito.ArgumentMatchers.anyMap;
 import static org.mockito.ArgumentMatchers.argThat;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
You'll need to adjust your IDE to avoid wildcard imports https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports